services/api/src/messaging/players.service.ts

```diff
@@ -84,21 +84,26 @@ const setDefaultStatusForPlayers = (ws: GameWebSocket): void => {
 export const clearBoard = (ws: GameWebSocket): void => {
   gameService.clearBoard(ws.roomId);
   setDefaultStatusForPlayers(ws);
+  sendMessage(ws, MessageType.SetMyTurn, null);
```

Comment:
why is this needed?

Reply:
Why do we need to have info about previous game. I think we just reset everything going into the next game.

Reply:
The same question remains. What is the reason behind having this piece of code when board is cleared? If its not necessary, we should remove it.